alvr/server_openvr/cpp/platform/linux/EncodePipeline.cpp

```diff
@@ -25,15 +25,15 @@ std::unique_ptr<alvr::EncodePipeline> alvr::EncodePipeline::Create(
     Renderer* render,
     VkContext& vk_ctx,
     VkFrame& input_frame,
-    VkFrameCtx& vk_frame_ctx,
+    VkImageCreateInfo image_create_info,
```

Comment:
You should pass this as a reference because copying around large structures is bad practice

Reply:
i wasn't sure what is best practice to copy or not CreateInfo structs, but if it's a bad practice, okay

Reply:
Generally `CreateInfo` structs contain pointers to other things, so whilst the pointers can still stay alive perfectly fine when copying the struct around, it creates a bit of a disconnect between everything.
So imo it's generally better to not copy `CreateInfo` structs, because then the actual objects behind the reference/pointers stay together in the place they were created.